API/events/triggerEvent.m

```diff
@@ -1,9 +1,15 @@
-function triggerEvent(eventType, data)
-    % Triggers the event type with the given data. The supported event types are
-    % 'message' and 'plot'. The data for message is a char array while for
-    % the plot it is a cell array containing the result and problem structs
+function triggerEvent(eventType, varargin)
+    % Triggers the event type with the given varargin. The supported event types are
+    % 0, 1, and 2. 
+    % * The input for the message event is a char array, 
+    % * The input for the plot event are the result cell, contrastParams.ssubs 
+    %   and problemStruct 
+    % * The input for progress events are the message (char array) and percentage 
+    %   value (double) btw 0 and 1.
```

Comment:
Worth writing "between" here

Reply:
Also, its a decimal value if its between 0 and 1. Maybe try ". . . and percentage progress expressed as a (double) decimal (i.e., between 0 and 1)."